config/config.msft.yaml

```diff
@@ -144,10 +144,9 @@ defaults:
   mise:
     deploy: true
     azureAdInstance: https://login.microsoftonline.com/
-    azureAdClientId: ""
+    azureAdClientId: 7f4a113a-c61d-412a-bea1-85dee5baf4a8 # ARO-HCP 1p App ID
```

Comment:
This is the FPA app ID but the property is named client ID. Is that intended?

Reply:
updated